04-Linking Accounts/app/controllers/settings_controller.rb

```diff
@@ -0,0 +1,47 @@
+class SettingsController < SecuredController
+  def show
+  @user = session[:userinfo]
+  @providers = user_providers.keys.collect {|x| [x,x]}
+  # [
+  #       ['Facebook','facebook'],
+  #       ['Github','github'],
+  #       ['Google','google-oauth2'],
+  #       ['Twitter','twitter']
+  # ]
+  end
+  def link_provider
+    user = session[:userinfo]
+    link_user = session[:linkuserinfo]
+
+    v2_creds = { client_id: ENV['AUTH0_CLIENT_ID'],
```

Comment:
I'll change the name of this variable as credentials. We don't need to have a difference in this example if its V1 or V2 api.